### ISSUE TITLE
why is WCAG 2 level A, a requirement, but AA not?

### ISSUE THREAD

**Issue by danielweck** [w3c/epub-specs#747]
http://www.idpf.org/epub/a11y/

CC @stevefaulkner

Steve Faulkner (Paciello Group) asked:

"why is WCAG 2 level A, a requirement, but AA not?"

https://twitter.com/accessibledaisy/status/753082359055089664

https://twitter.com/stevefaulkner
https://github.com/stevefaulkner

https://github.com/thepaciellogroup
https://www.paciellogroup.com/

**Comment by mattgarrish** [comment]
The recommendation is to meet level AA, but just as level A exists in WCAG we aren't excluding content that conforms. The note immediately following the requirement (which we're going to turn back into plain spec prose) explains:

> Local and national laws can influence the level of conformance an EPUB Publication has to meet to be considered accessible, as [WCAG 2.0] Level AA conformance is often cited as the benchmark for accessibility in legal frameworks and policies. In fact, any EPUB specification, and even vendor procurement requirements, can demand content that exceeds the minimum requirements of this specification. Authors need to ensure they understand the requirements their content has to meet to be considered accessible in any jurisdiction or distribution context, including seeking any necessary legal advice. Minimum conformance with this specification does not supersede such requirements, and does not offer protection from any legal liability that might arise.

An EPUB Publication also has to indicate what level of WCAG conformance it has achieved in the metadata, to prevent different quality publications from being blurred together.

Does that answer the question?

**Comment by mattgarrish** [comment]
Closing for lack of further follow-up.
